src/app/components/ThemeProvider/fontFacesLazy.ts

```diff
@@ -1,30 +1,24 @@
 import { Services } from '#app/models/types/global';
 
-const REITH_FONTS_DIR = 'https://static.files.bbci.co.uk/fonts/reith/r2.512/';
+const FONTS_BASE_URL = 'https://static.files.bbci.co.uk/fonts/';
```

Comment:
Fonts live in 2 different places.

Reith & Reith Qalam are in https://static.files.bbci.co.uk/fonts
World Service ones are in https://static.files.bbci.co.uk/ws/simorgh-assets/public/fonts

So I think we need to have the following: 

```suggestion
const REITH_BASE_URL = 'https://static.files.bbci.co.uk/fonts/';
const REITH_FONTS_DIR = `${REITH_BASE_URL}reith/r2.512/`;
const REITH_QALAM_FONTS_DIR = `${REITH_BASE_URL}reith-qalam/1.310/`;

const FONTS_BASE_URL = 'https://static.files.bbci.co.uk/ws/simorgh-assets/public/fonts/';
```

Reply:
Ok, done